src/components/Modals/Send/AddressInput/AddressInput.tsx

```diff
@@ -35,7 +35,13 @@ export const AddressInput = ({ rules }: AddressInputProps) => {
             spellCheck={false}
             autoFocus
             fontSize='sm'
-            onChange={e => onChange(e.target.value.trim())}
+            onChange={e => {
+              if (e.target.value.trim() === '') {
+                setValue(SendFormFields.Address, '')
+              }
```

Comment:
There was an issue where if you setup a valid address, then remove it completely, you can continue the flow because the `Address` form state wasn't empty

Reply:
let's handle this stuff in a separate PR, as this doesn't fix the bug with cosmos, and it keeps this PR related to just eip681